### ISSUE TITLE
Pin image versions

### ISSUE THREAD

**Issue by Karinon** [FREVA-CLINT/freva-deployment#65]
I am using a linter for Dockerfiles in my Neovim, which gave me the following warning in a random Freva-related Dockerfile:

```
Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag
```

This is out-of-the-box behavior, just so you don't get the feeling I manipulated it to meet my anti-bleeding-edge agenda :P

I agree with this warning as I already have experienced broken containers because the image was updated. It would make the whole system more reliable.

**Comment by antarcticrainforest** [comment]
I have added tests that build the container and do some simple testing the if container is working. In addition with the unit tests this should be safe - if we have unit tests.

**Comment by antarcticrainforest** [comment]
Addressed in #105
